addons/website/views/snippets/s_ecomm_categories_showcase.xml

```diff
@@ -0,0 +1,157 @@
+                    </div>
+                </div>
+                <div 
+                    class="s_ecomm_categories_showcase_block o_cc o_cc5 oe_img_bg rounded-2"
+                    data-name="Block"
+                    style="background-image: url('/web/image/website.s_ecomm_categories_showcase_3');"
+                    >
+                    <div class="o_we_bg_filter bg-black-25"></div>
+                    <div class="s_ecomm_categories_showcase_row align-items-end">
+                        <div class="s_ecomm_categories_showcase_content text-start">
+                            <h3>Drawers</h3>
+                            <a href="" title="" role="button" class="btn btn-secondary">Shop All</a>
+                        </div>
+                    </div>
+                </div>
+                <div 
+                    class="s_ecomm_categories_showcase_block o_cc o_cc5 oe_img_bg rounded-2"
+                    data-name="Block"
+                    style="background-image: url('/web/image/website.s_ecomm_categories_showcase_4');"
+                    >
+                    <div class="o_we_bg_filter bg-black-25"></div>
+                    <div class="s_ecomm_categories_showcase_row align-items-end">
+                        <div class="s_ecomm_categories_showcase_content text-start">
+                            <h3>Lamps</h3>
+                            <a href="" title="" role="button" class="btn btn-secondary">Shop All</a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+</template>
+
+<template id="s_ecomm_categories_showcase_snippet_options" inherit_id="website.snippet_options">
+    <xpath expr="//t[@t-call='website.snippet_options_background_options']" position="before">
+        <div data-selector=".s_ecomm_categories_showcase"
+            data-js="BlockCount"
+            >
+            <we-select 
+                string="# of Blocks" 
+                data-attribute-name="blockCount" 
+                data-attribute-default-value="4"
+                >
+                <we-button data-select-data-attribute="2">2</we-button>
+                <we-button data-select-data-attribute="3">3</we-button>
+                <we-button data-select-data-attribute="4">4</we-button>
+            </we-select>
+        </div>
+        <div 
+            data-js="vAlignment"
+            id="s_ecomm_categories_showcase_valign_snippet_option"
+            data-selector=".s_ecomm_categories_showcase_block"
+            data-target=".s_ecomm_categories_showcase_row"
+            >
+            <t t-call="website.vertical_alignment_option"/>
+        </div>
+        <div 
+            id="s_ecomm_categories_showcase_text_align"
+            data-selector=".s_ecomm_categories_showcase_block"
+            data-target=".s_ecomm_categories_showcase_content"
+            >
+            <we-button-group string="Text Alignment">
```

Comment:
It works.. until it doesn't 😕
It "conflicts" with the editor; indeed if you use the editor controller, this option will not apply anymore (or at least not consistently).

I think we have no other option than remove it 😕

| |
|--------|
| ![image](https://github.com/user-attachments/assets/49dd029c-1840-4fd6-bddf-8c762ab3f7e1) |